### PR TITLE
add the basic arithmetic operation

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
Add arithmetic operations:
addition
subtraction
multiplication
division
mod

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,39 +1,131 @@
...
+          </button>
+          <button type="button" className="button calc-equal" onClick={this.ResultHandler}>
+            <div className="cell">=</div>
+          </button>
         </div>
       </div>
-
     );
   }
 }
```
Comment (andersonlebon) [inline src/components/Calculator.js:131]: - [ ] Kindly note that it's really required to use the calculation logic provided in the requirement. see [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js)

File: src/components/operate.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return Number(state.firstNumber) * Number(state.secondNumber);
+  } if (state.operation === '÷') {
+    return Number(state.firstNumber) / Number(state.secondNumber);
+  } if (state.operation === '%') {
+    return Number(state.firstNumber) % Number(state.secondNumber);
+  }
+  return 0;
+}
+
+export default { readEqual };
```
Comment (andersonlebon) [inline src/components/operate.js]: - [ ] Kindly note that it's really required to use the operation logic provided by the requirement. see [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @maenkhraisha 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,76 +1,45 @@
...
-        firstNumber: '', secondNumber: '', operation: '',
-      },
-    );
+  CalculatorHandler(buttonName) {
+    const result = calc(this.state, buttonName);
+    this.setState(() => ({
+      total: result.total,
+      next: result.next,
+      operation: result.operation,
+    }));
```
Comment (medaminedev66) [inline src/components/Calculator.js:26]: [OPTIONAL] Class methods are written in camelCase, it’s a best practice to actually tell what the function is doing by giving the function name a verb as prefix.
```js

// bad
name(firstName, lastName) {
  return `${firstName} ${lastName}`;
}

// bad
Name(firstName, lastName) {
  return `${firstName} ${lastName}`;
}
 
// good
getName(firstName, lastName) {
  return `${firstName} ${lastName}`;
}
```

**Review by medaminedev66 — APPROVED** [review]
Hi @maenkhraisha, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**